tests/Oryx.Integration.Tests/LocalDockerTests/Fixtures/MySqlDbContainerFixture.cs

```diff
@@ -37,19 +37,26 @@ protected override DockerRunCommandResult RunDbServerContainer()
 
         protected override bool WaitUntilDbServerIsUp()
         {
-            // Try 30 times at most, with a constant 2s in between attempts
-            var retry = Policy.HandleResult(result: false).WaitAndRetry(30, i => TimeSpan.FromSeconds(2));
+            var retry = Policy.HandleResult(result: false).WaitAndRetry(retryCount: 36, i => TimeSpan.FromSeconds(5));
             return retry.Execute(() =>
             {
-                // Based on https://hub.docker.com/r/mysql/mysql-server/#starting-a-mysql-server-instance
-                string status = _dockerCli.GetContainerStatus(DbServerContainerName);
-                return status.Contains("healthy") && !status.Contains("starting");
+                try
+                {
+                    // Based on https://hub.docker.com/r/mysql/mysql-server/#starting-a-mysql-server-instance
+                    string status = _dockerCli.GetContainerStatus(DbServerContainerName);
+                    return status.Contains("healthy") && !status.Contains("starting");
+                }
+                catch
+                {
+                    // In case of any exception, we consider this retry as a failure and will retry again.
+                    return false;
+                }
             });
         }
 
         protected override void InsertSampleData()
         {
-            const string sqlFile = "/tmp/setup.sql";
+            const string sqlFile = "/tmp/mysql_setup.sql";
```

Comment:
Each database type has this method of its own and all were using the same file location on the agent, so a potential race condition where a sql script intended for one database could have been used for another (since database types run in parallel for different database server types)